Client/Ecosia/L10N/en.lproj/Ecosia.strings

```diff
@@ -231,3 +231,13 @@
 "Customizable home page" = "Customizable home page";
 "Tailor your home page to show the information that’s most relevant to you." = "Tailor your home page to show the information that’s most relevant to you.";
 "Discover" = "Discover";
+
+// APN Consent
```

Comment:
Oh, now the changes appear again 🤔 I guess the issue is only on transifex's PRs? Or maybe only on non-english files? The plot thickens 🤯